pandas/tests/frame/test_analytics.py

```diff
@@ -593,7 +593,7 @@ def test_kurt(self):
                     "E": Categorical(["a"], categories=["a"]),
                     "F": to_datetime(["2000-1-2"]),
                     "G": to_timedelta(["1 days"]),
-                },
+                }
```

Comment:
Multiline statements such as

````
{
    'a': 1,
    'b': 2,
}
````

should keep the trailing comma; it prevents having an extra line in the diff if a new entry is added. For the black upgrade, we're looking to remove this trailing comma when it occurs on a single line:

    {'a': 1, 'b': 2}  # rather than {'a': 1, 'b': 2,}